src/test/functional/features/email-addresses.feature

```diff
@@ -12,37 +12,37 @@ Feature: Email-addresses
     When I click the Emails tab
     Then I can view the existing emails
 
-  Scenario Outline: Add new Email
-    And I add Description from the dropdown "<adminId>" and Address "<address>" and Explanation "<explanation>" and Welsh Explanation "<explanationCy>"
-    When I click on Add new Email
+  Scenario: Add new Email Addresses
+    When I add Description from the dropdown at index 6 and Address "abs@gmail.com" and Explanation "County Court" and Welsh Explanation "Llys sirol"
+    Then I click on Add new Email
+    And I add Description from the dropdown at index 8 and Address "functional.test1@testing.com" and Explanation "Testing - English" and Welsh Explanation "Testing - Welsh"
     And I click save button
     Then a green update message showing email updated is displayed
+    And the second last email address is displayed with description at index 6 Address "abs@gmail.com" Explanation "County Court" and Welsh Explanation "Llys sirol"
+    And the last email address is displayed with description at index 8 Address "functional.test1@testing.com" Explanation "Testing - English" and Welsh Explanation "Testing - Welsh"
 
-    Examples:
-      | adminId | address       | explanation  | explanationCy |
-      | 6       | abs@gmail.com | County Court | Llys sirol    |
-
-  Scenario: Leave email type and address blank
-    When I click on Add new Email
+  Scenario: Incomplete email type and address
     When I leave adminId blank
-    And I leave Address blank
-    When I click on Add new Email
+    And I add address "incomplete@test.com"
     And I click save button
     Then A red error message display
 
   Scenario: Remove emails
     When I click the remove button below a email section
     And I click save button
     Then a green update message showing email updated is displayed
+    # Delete another to remove both additions from above and test multiple deletes
```

Comment:
Not sure if you still want to keep this comment in @agraham01 ?

Reply:
I've removed the comment